sys/net/crosslayer/ng_netif/ng_netif.c

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ */
+
+#include <stdio.h>
+
+#include <errno.h>
+#include "kernel_types.h"
+#include "net/ng_netif.h"
+
+static ng_netif_handler_t if_handler[] = {
```

Comment:
I am not sure if we need this handlers. To add/set/rem e.g. an IPv6 address for a device, I would do it the following way (pseudo code):
```
dev_pid = ...;    // PID of the network device
addr = {};        // IPv6 address to add to the device
// get the PID for the IPv6 protocol
ip_pid = netreg_lookup(NETTYPE_IPV6);
// then I set a new address using netapi
netapi_set(ip_pid, NETCONF_OPT_ADDRESS, dev_pid, &addr, sizeof(addr));
```
@authmillenon Do you agree with this?